fastlane_bot/events/async_event_update_utils.py

```diff
@@ -536,3 +544,48 @@ def async_update_pools_from_contracts(mgr: Any, current_block: int, logging_path
     mgr.cfg.logger.info(
         f"Async Updating pools from contracts took {(time.time() - start_time):0.4f} seconds"
     )
+
+
+def update_remaining_pools(mgr):
+    remaining_pools = []
+    all_events = [
+        event
+        for _, _, event, _, _ in mgr.pools_to_add_from_contracts
+    ]
```

Comment:
More readable perhaps:
```
all_events = [pool[2] for pool in mgr.pools_to_add_from_contracts]
```
BTW, is each one of these "pools" guaranteed to always have at least 3 elements (or if you intend to keep your current version of the code - at least 5 elements)?

Reply:
changed as you suggest